### PR TITLE
Added logs for content installation

### PR DESCRIPTION
**Pull request by hod-alpert**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
Related: https://github.com/demisto/etc/issues/30511

## Description
* Implemented logging in `Install Content And Configure Integrations On Server`
* Removed `printsManager` from where possible (All functions that are not being used in `Run Tests` step too)
* Added a `install_simple_logging` function that only configures the success logging level, the colors and the log format (message only) so that when replacing it with print_error, print_warning etc. will behave the same for the common functions with  `Run Tests`.
* It appears the sdk validator imports a package that calls logging basicConfig method before the log_util function.
Therefore, the `install_logging` method is executed before the sdk Validator is imported to enforce the correct logging configurations.
* Added debug logs and modified info logs where ever seems right

## Special build examples:
* Nightly: https://app.circleci.com/pipelines/github/demisto/content/42965/workflows/6c4b8bb6-8006-4d98-a1c8-9bd50bc216d5
* Instance Testing:  https://app.circleci.com/pipelines/github/demisto/content/42739/workflows/953cbfbf-a6e4-4e67-b214-a5977e9fafd4
* Private build: https://github.com/demisto/content-private/runs/1390923024

### CODE REVIEW

File: Tests/configure_and_test_integration_instances.py [hunk 1]
```diff
@@ -1066,76 +1038,65 @@ def get_pack_ids_to_install():
...
-        integrations_names = [i.get('name') for i in integrations]
-        prints_manager.add_print_job('All Integrations for test "{}":'.format(test.get('playbookID')), print_warning, 0)
-        prints_manager.add_print_job(integrations_names, print_warning, 0)
+        playbook_id = test.get('playbookID')
 
         new_integrations, modified_integrations, unchanged_integrations, integration_to_status = group_integrations(
             integrations, build.skipped_integrations_conf, all_new_integrations, modified_integrations
         )
-
+        integration_to_status_string = '\n\t\t\t\t\t\t'.join(
```
Comment (yaakovi) [inline Tests/configure_and_test_integration_instances.py:1090]: why did you add all the tabs only here?
you don't do it in any other multiline log

File: Tests/test_integration.py [hunk 2]
```diff
@@ -593,29 +619,35 @@ def __create_integration_instance(server, username, password, integration_name,
...
+        logging.debug(f'Disabling integration {module_instance.get("name")}')
         try:
             res = demisto_client.generic_request_func(self=client, method='PUT',
                                                       path='/settings/integration',
                                                       body=module_instance)
         except ApiException as conn_err:
-            error_message = 'Failed to disable integration instance, error trying to communicate with demisto ' \
-                            'server: {} '.format(conn_err.body)
-            prints_manager.add_print_job(error_message, print_error, thread_index)
+            if prints_manager:
```
[the remark this answers is not in the document]
Comment (hod-alpert) [reply Tests/test_integration.py:636]: Fixed here https://github.com/demisto/content/pull/9945

File: Tests/scripts/utils/log_util.py [hunk 3]
```diff
@@ -96,3 +96,21 @@ def install_logging(log_file_name: str, include_process_name=False) -> str:
     logging.basicConfig(level=logging.DEBUG,
                         handlers=[ch, fh])
     return log_file_path
+
+
+def install_simple_logging():
```
[the remark this answers is not in the document]
Comment (hod-alpert) [reply Tests/scripts/utils/log_util.py:101]: Because I don't want to write to a logging file yet until it's fully implemented
And because I don't want to change the current behavior.

This is only to allow me to replace the print_error, print_warning, print_color methods with this one without any changes to the output.

File: Tests/configure_and_test_integration_instances.py [hunk 4]
```diff
@@ -11,27 +10,33 @@
...
-from Tests.test_content import extract_filtered_tests, ParallelPrintsManager, \
-    get_server_numeric_version
+from Tests.test_content import extract_filtered_tests, get_server_numeric_version
 from Tests.update_content_data import update_content
 from Tests.Marketplace.search_and_install_packs import search_and_install_packs_and_their_dependencies, \
     install_all_content_packs, upload_zipped_packs
-
 from Tests.tools import update_server_configuration
+if __name__ == '__main__':
+    # The ValidateManager class imports a package that's configuring logging's basicConfig.
```
[the remark this answers is not in the document]
Comment (hod-alpert) [reply Tests/configure_and_test_integration_instances.py:35]: No, but handled it better.
logging basicConfig has a force flag to overwrite previous configurations.
With that we can go back to the original design. where logging is installed in the main function

File: Tests/configure_and_test_integration_instances.py (hunk 1 above)
Comment (hod-alpert) [reply Tests/configure_and_test_integration_instances.py:1090]: All the other multiline logs happens only once.
these logs haapen as the number of packs that should be installed and it makes the logs a bit confusing.
Look at the example logs and tell me what you think..

**Review by yaakovi — APPROVED** [review]
